### DISCUSSION TITLE
Question about "Not in an actively usable state"

### DISCUSSION THREAD

**Discussion by KoolenDasheppi** [EdNekebno/DGTCentaurMods#8] [General]
When you say "not in an actively usable state", what do you mean exactly? Do you mean it's non-functional? Or do you mean it still needs to be worked on? If so, what kind of state is the project in right now? I'd like to try it out when my DGT Centaur comes in. I have a Pi Zero W and everything. I'd love to have lichess integration on my board, it sounds amazing. Also, is it possible to swap out the SD cards? So I could have both a lichess version and the stock functionality whenever I swap them out? Thanks in advance.

**Comment by EdNekebno** [comment]
I mean there's still some work to be done on the actual scripts (lichess, for example, sometimes misdetects and doesn't handle pawn promotion) and that the whole thing is not in a form that you can easily install it. Basically it's a hint that it's alpha and you'll have to tinker quite a bit yourself. For a lot less tinkering/playing around it might be worthwhile waiting on this fork: https://github.com/dsommerfeld/DGTCentaur-2.0-by-dso that's working on some of these things (time is a bit short for me right now). Their idea is then to give an sd card image that you can burn to another sd card and just swap into pi.  (the menu.py can by the way also call the stock program. See https://twitter.com/ednekebno/status/1384861881744101376 ).

**Comment by KoolenDasheppi** [comment]
Wow, thanks for such a fast reply! I'll look into the fork, and also try setting this one up too maybe. I didn't think that this project conserved the stock functionality to be honest, but that's awesome that it does. Now all I have to do is wait for my board to get here tomorrow.

**Comment by javiram10** [comment]
Wow! Fantastic job! I created my own electronic chess and programmed it in Objective C and Arduino, although hall sensors break every other day and magnets do not work flawlessly. I have been thinking about purchasing the DGT Centaur but I do not want to do the investment until knowing I can make the board play in Stockfish levels that are not adaptive, so I can track my improvement. I heard the process of rooting the DGT Centaur is not as easy as in the first boards... Can any of you confirm? I will then look at the Python code and try to figure things out to buy it and start the substituting of the Raspberry Pi, SD Card and coding. Thank you!

**Comment by EdNekebno** [comment]
> 
> 
> Wow! Fantastic job! I created my own electronic chess and programmed it in Objective C and Arduino, although hall sensors break every other day and magnets do not work flawlessly. I have been thinking about purchasing the DGT Centaur but I do not want to do the investment until knowing I can make the board play in Stockfish levels that are not adaptive, so I can track my improvement. I heard the process of rooting the DGT Centaur is not as easy as in the first boards... Can any of you confirm? I will then look at the Python code and try to figure things out to buy it and start the substituting of the Raspberry Pi, SD Card and coding. Thank you!

Sorry for the slow reply. I have no idea in comparison to the first boards. I don't remember it being any hard than a standard root: https://raspberrypi.stackexchange.com/questions/98353/forgot-password-for-username-pi . The card with the pi has a very limited set of packages installed though. It gets restrictive/complicated when you want to do or install more.
